rules-reviewed/eap7/eap4and5/jboss-eap4and5to6and7-xml.windup.xml

```diff
@@ -127,7 +126,7 @@
                 <hint issue-display-mode="all" title="Create org.jboss.naming.NamingAlias class" effort="1" category-id="mandatory">
                     <message>
                         <![CDATA[
-                        JBoss EAP 6 supports deploying SAR files, however JBoss EAP 5 and 6 do not share a lot of the internals.  
```

Comment:
is this double space removal on purpose ? we put it to create a new line after the text.